src/index.html

```diff
@@ -4,10 +4,408 @@
+                      alt="Audio sunglasses"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+                <div class="grid__item grid__item--desktop-5-12 grid__item--tablet-3-6"> 
+                  <div class="place__photo-container">
+                    <img
+                      src="./images/user7.png"
+                      alt="Audio sunglasses"
+                      class="place__photo place__photo--six"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="place__description place__description--margin">
+              Audio sunglasses
+            </div>
+          </div>
+          <div class="button">
+            <div class="grid grid--mobile">
+              <div class="grid__item grid__item--desktop-5-8 grid__item--tablet-2-5">
+                <a href="#gallery" class="button__link">
+                  View all
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="testimonials page__section" id="testimonials">
+        <div class="container">
+          <div class="grid grid--mobile">
+            <div class="grid__item grid__item--tablet-1-3 grid__item--desktop-1-4">
+              <h2 class="page__section-title testimonials__title">
+                Why buy direct
+                <br>
+                from Bose
+              </h2>
+            </div>
+            <div class="grid__item  grid__item--tablet-4-6 grid__item--desktop-6-12">
+              <h3 class="page__section-subtitle testimonials__subtitle">
+                Free 2-day shipping and returns
+              </h3>
+
+              <h3 class="page__section-subtitle testimonials__subtitle">
+                90-day risk-free trial
+              </h3>
+
+              <h3 class="page__section-subtitle testimonials__subtitle">
+                World class customer service
+              </h3>
+
+              <h3 class="page__section-subtitle testimonials__subtitle">
+                My Bose account management
+              </h3>
+
+              <p class="page__section-text testimonials__text grid__item grid__item--tablet-4-6 grid__item--desktop-6-10">
+                A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="contacts page__section" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--mobile">
+            <div class="grid__item grid__item--desktop-1-5 grid__item--tablet-1-3">
+              <form
```

Comment:
<img width="682" alt="Monosnap Miami 2022-11-23 09-40-41" src="https://user-images.githubusercontent.com/39308717/203493980-a39857ea-4012-45b7-84df-e0ac7e8a01e0.png">

Reply:
I could not manage to erase it

Reply:
are you sure @dolnaro ?
tried line 41 of `styles\blocks\contacts.scss`?
<img width="600" alt="Monosnap Miami 2022-11-23 09-58-29" src="https://user-images.githubusercontent.com/39308717/203496436-584ebe8e-61bc-4008-896b-baa6ba905be2.png">